src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/ContextIntervalTree.cs

```diff
@@ -150,7 +157,7 @@ private T GetSmallestContainingIntervalWorker(int start, int length, Func<T, int
                         {
                             // right side tree doesn't have any answer or if the right side has
                             // an answer but left side can have better answer then try left side
-                            if (EqualityComparer<T>.Default.Equals(result, default) ||
+                            if (EqualityComparer<T/*??*/>.Default.Equals(result!, default!) ||
```

Comment:
i wonder if we should have a static for this, typed to be null-forgiving, which just trasparently exposes the true underlying comparer.

Reply:
We haven't needed this frequently, and I think `??` will be coming at some point.

Reply:
I can file an issue for it.

Reply:
➡️ https://github.com/dotnet/roslyn/issues/43390